app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt

```diff
@@ -602,7 +602,10 @@ class DirectoryAdapter(
                 } else {
                     val paths = getSelectedPaths()
                     val fileDirItems = ArrayList<FileDirItem>(paths.size)
-                    paths.forEach { fileDirItems.add(File(it).toFileDirItem(activity)) }
+                    paths.forEach {
+                        val fileDirItem = FileDirItem(it, it.getFilenameFromPath(), activity.getIsPathDirectory(it))
+                        fileDirItems.add(fileDirItem)
+                    }
                     val size = fileDirItems.sumByLong { it.getProperSize(activity, countHidden = true) }.formatSize()
```

Comment:
Changed back to three lines for readdability

Reply:
Thats 4... I mean that you should still use File(it).toFileDirItem(activity), just add new lines around brackets

Reply:
Am sorry for the mistake I changed back to File(it).toFileDirItem(activity) in the last commit